File: server/models/contract.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;

var contract = new Schema({
    movieId: { type: String, required: true },
    typeOfCont: { type: Number, required: true },
    firstSide: { type: String, required: true },
    secondSide: { type: String, required: true },
    contNum: { type: String, required: true },
    condition: { type: Number, required: true },
    tax: { type: Number },
    condPercent: { type: Boolean, required: true },
    contDate: { type: String, required: true },
    fromDate: { type: String, required: true },
    toDate: { type: String, required: true },
    parentId: { type: String },
    // prices
    dayChildPriceTh: {type: String},
    dayAdultPriceTh: {type: String},
    eveningChildPriceTh: {type: String},
    eveningAdultPriceTh: {type: String},
    dayChildPriceGr: {type: String},
    dayAdultPriceGr: {type: String},
    eveningChildPriceGr: {type: String},
    eveningAdultPriceGr: {type: String},
    dayChildPriceMobile: {type: String},
    dayAdultPriceMobile: {type: String},
    eveningChildPriceMobile: {type: String},
    eveningAdultPriceMobile: {type: String}
});
module.exports = mongoose.model('contract', contract);

File: server/androidClient/app/src/main/java/com/example/kinoprokat/models/Contract.java
package com.example.kinoprokat.models;

import java.util.Date;

public class Contract {

    private String _id;
    private String movieId;
    private String firstSide;
    private String secondSide;
    private String contNum;
    private Date fromDate;
    private Date toDate;
    private int typeOfCont;
    private String movieUser;

    // theater prices
    private int dayChildPriceTh;
    private int dayAdultPriceTh;
    private int eveningChildPriceTh;
    private int eveningAdultPriceTh;
    // mobile theaters prices
    private int dayChildPriceMobile;
    private int dayAdultPriceMobile;
    private int eveningChildPriceMobile;
    private int eveningAdultPriceMobile;


    public String get_id() {
        return _id;
    }

    public String getMovieId() {
        return movieId;
    }

    public String getFirstSide() {
        return firstSide;
    }

    public String getSecondSide() {
        return secondSide;
    }

    public Date getFromDate() {
        return fromDate;
    }

    public Date getToDate() {
        return toDate;
    }

    public int getTypeOfCont() {
        return typeOfCont;
    }

    public String getMovieUser() {
        return movieUser;
    }

    public String getContNum() {
        return contNum;
    }

    public int getDayChildPriceTh() {
        return dayChildPriceTh;
    }

    public int getDayAdultPriceTh() {
        return dayAdultPriceTh;
    }

    public int getEveningChildPriceTh() {
        return eveningChildPriceTh;
    }

    public int getEveningAdultPriceTh() {
        return eveningAdultPriceTh;
    }

    public int getDayChildPriceMobile() {
        return dayChildPriceMobile;
    }

    public int getDayAdultPriceMobile() {
        return dayAdultPriceMobile;
    }

    public int getEveningChildPriceMobile() {
        return eveningChildPriceMobile;
    }

    public int getEveningAdultPriceMobile() {
        return eveningAdultPriceMobile;
    }
}
